### PR TITLE
fix description of sharing toggle

### PR DESCRIPTION
**Pull request by szaimen**
| Before | After |
|---|---|
| image | image |

### CODE REVIEW

File: apps/files_sharing/src/actions/sharingStatusAction.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ export const action = new FileAction({
 			return t('files_sharing', 'Shared')
 		}
 
-		return ''
+		return t('files_sharing', 'Show sharing options')
```
Comment (szaimen) [inline apps/files_sharing/src/actions/sharingStatusAction.ts:57]: @skjnldsv this on one hand makes sure that the item has a desciption in the dropdown:
image
However it also displays inline which we probably do not want:
image
Do you have an idea how to fix both at the same time?

**Review by skjnldsv — APPROVED** [review]

**Review by artonge — APPROVED** [review]

**Comment by szaimen** [comment]
Unfortunately it does not work in gird view:
image

**Comment by szaimen** [comment]
I guess this is the problem?
image

**Review by susnux — APPROVED** [review]

**Pull request merged**